src/App.jsx

```diff
@@ -1,9 +1,138 @@
-import React from "react";
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+
+import "./Apps/Login.css";
 
 export const App = () => {
-    return (
-        <div>
-            Atakan Ayyıldız
+  // React States
+  const [errorMessages, setErrorMessages] = useState({});

```

Comment:
Burada bizim react hook form kullanabiliriz. Hem şirket için iyi çok önemli bir pratik olur senin için.